tests/Microsoft.Fx.Portability.MetadataReader.Tests/ReflectionMetadataInfoTests.cs

```diff
@@ -44,17 +44,16 @@ public void UnresolvedAssemblyTest()
 
         private IEnumerable<string> _expectedResult = new[]
         {
-            "xunit.assert, Version=2.1.0.2945, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c",
-            "NSubstitute, Version=1.8.1.0, Culture=neutral, PublicKeyToken=92dd2e9066daa5ca",
+            "Microsoft.CodeAnalysis, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35",
```

Comment:
Does this mean that you'll have to update this every time we pick up a newer drop?